frontend/packages/dev-console/integration-tests/support/pages/add-flow/add-page.ts

```diff
@@ -10,61 +10,61 @@ export const addPage = {
     switch (card) {
       case 'Git':
       case addOptions.Git:
-        cy.byLegacyTestID('import-from-git').click();
+        cy.byLegacyTestID('odc-add-card-item-import-from-git').click();
```

Comment:
We will discuss this test ids in a meeting later today. After that I will also update https://github.com/openshift/console/pull/8998 based on our decision. This will require that we update this times here (or maybe we can just keep the old data test ids here).